### DISCUSSION TITLE
Restricted accounts top plays being invisible to that user

### DISCUSSION THREAD

**Discussion by AosUK** [ppy/osu#23699] [Ideas]
On the osu website, top plays are visible for restricted accounts:
![website_best_scores](https://github.com/ppy/osu/assets/134084802/548b648f-5a67-48d6-a113-406dddd1562e)

However this is not the case in lazer:

![lazer_best_scores](https://github.com/ppy/osu/assets/134084802/be62be8c-ee88-4834-9bc4-c6b7a9e5aa24)

I find this especially strange as recent scores ARE shown in lazer:
![lazer_recent_scores](https://github.com/ppy/osu/assets/134084802/1bad275b-c850-44dc-8f3c-38091372e42f)

I see hardly any reason why to hide the top plays from the user themselves, after all it affects nobody else.

--------
I also have thought about the possibility of showing what rank(s) a player would be (again, only to them) if they were not restricted.
once again I don't see a reason to not do this as its only visible for them and will not interfere with other players.
-maybe just an estimate of this as determining the exact leaderboard position may not be possible without large recalculations.

![lazer_player_ranking](https://github.com/ppy/osu/assets/134084802/fd95f56f-af87-4324-bc70-69fd673d0277)

but I do see the argument that it doesn't really make sense to compare yourself to a leaderboard you don't contribute to, but regardless I think these features would be nice and hopefully not too hard to implement

**Comment by peppy** [comment]
Probably going to require an osu-web change @ppy/team-web.
